app/assets/stylesheets/provider/admin/_dashboard.scss

```diff
@@ -71,12 +71,6 @@
   }
 }
 
-// DashboardWidget
-// this should be encapsulated within individual widget.
-.DashboardSection--wide .DashboardWidget {
-  width: 100%;
-}
-
```

Comment:
Why remove this? I think this is valid in some provider member use cases

Reply:
Ah, I should've extracted into a different PR. With the new widgets, it's no longer useful.